CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 cmake_minimum_required(VERSION 2.8.12)
-
+if (POLICY CMP0074)
+  cmake_policy(SET CMP0074 NEW)
```

Comment:
This allows me to set the environment variable `FLINT_ROOT` for example. I think this is rather harmless to enable, no?